projects/shared/src/lib/view/thumbnail/thumbnail.component.scss

```diff
@@ -21,7 +21,7 @@ img {
   width: 80px;
 }
 
-@media (max-width: 960px) {
+@media (max-width: 768px) {
```

Comment:
Do you think it's possible to set size with `%` or `rem` or something relative?

Reply:
That codes re-uses what is set on rero-ils. It adapts the size of the cover image according to the screen size. I don't see a real improvement by using relative values.

Reply:
Ok.